nav2_waypoint_follower/CMakeLists.txt

```diff
@@ -59,15 +59,14 @@ set(dependencies
   OpenCV
 )
 
-ament_target_dependencies(${executable_name}
```

Comment:
Here, it's better to link the dependencies to the library, then link the library to the executable, that way you guarantee the library is not missing anything.